com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/messages.properties

```diff
@@ -35,6 +35,7 @@ error_missing_properties = Invalid type definition, object has a required field
 warning_simple_reference = Simplified reference syntax is deprecated. The reference should be a valid JSON pointer.
 error_array_missing_items= Invalid array definition, items type should be present
 error_invalid_reference_type = Invalid object reference, the referenced object is not of expected type
+error_invalid_parameter_location = Invalid parameter location value, possible values are \"query\", \"header\", \"path\" or \"cookie\".
```

Comment:
I just noticed that `OpenApi3Validator` from `com.reprezen.swagedit.openapi3` uses `Messages` from `com.reprezen.swagedit.core`. 
It's OK for now, but we should fix it.